apache/superset
# docs: add doc for maxcompute support

### SUMMARY
This pull request adds documentation on maxcompute support. MaxCompute is a serverless data warehouse solution provided by Alibaba Cloud.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
(not applicable)

### TESTING INSTRUCTIONS
Pulled up a local superset instance and connected to my maxcompute instance to make sure it connects and is able to generate charts / run sqls. 

### ADDITIONAL INFORMATION
